email/parse.c

```diff
@@ -74,8 +74,7 @@ void mutt_auto_subscribe(const char *mailto)
   {
     const char *mailbox = TAILQ_FIRST(&lpenv->to)->mailbox;
     if (mailbox && !mutt_regexlist_match(&UnSubscribedLists, mailbox) &&
-        !mutt_regexlist_match(&UnMailLists, mailbox) &&
-        !mutt_regexlist_match(&UnSubscribedLists, mailbox))
+        !mutt_regexlist_match(&UnMailLists, mailbox))
```

Comment:
I've checked through the history (and upsteam code) and it seems it's a typo.
The first should be `SubscribedLists`:
```c
    if (mailbox && !mutt_regexlist_match(&SubscribedLists, mailbox) &&
        !mutt_regexlist_match(&UnMailLists, mailbox) &&
        !mutt_regexlist_match(&UnSubscribedLists, mailbox))
```